scaling_performance/cns_guidance.adoc

```diff
@@ -0,0 +1,82 @@
+[[scaling-performance-cns-guidance-for-databases]]
+= CNS Guidance For Databases
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[CNS-Guidance-For-Databases]]
+== CNS Guidance for Databases
+In this document will be listed best practices and guidance we recommend 
+for case when CNS ( Container Native Storage ) is used as storage for applications. 
+
+[[Tested-Applciations]]
+== Tested Applications 
+In OCP v3.10 exensive testing was done with (no)SQL databases
+
+-  Postgresql SQL
+-  Mongodb noSQL 
+
+As storage for databases were used storage originating from CNS storage cluster. 
+
+For Postgresql benchmarking link:https://www.postgresql.org/docs/10/static/pgbench.html[pgbench] was used for database benchmarking.
+For noSQL MongoDB benchmarking YCSB link:https://github.com/brianfrankcooper/YCSB/tree/master/mongodb[Yahoo! Cloud Serving Benchmark] was used for benchmarking and link:https://github.com/brianfrankcooper/YCSB/tree/master/workloads[workloada,workloadb,workloadf] were tested
+
+[[Support-Matrix]]
+== Support Matrix 
+
+.Table Title - CNS File 
+|===
+|Database | CNS File |Turn *off* Performance Translators |Turn *on* Performance Translators
+|Postgresql
+|Yes
+a| 
+- performance.stat-prefetch 
+- performance.read-ahead  
+- performance.write-behind 
+- performance.readdir-ahead 
+- performance.io-cache 
+- performance.quick-read 
+- performance.open-behind 
+a| 
+- performance.strict-o-direct
+
+|MongoDB 
+|Yes
+a|
+- performance.stat-prefetch 
+- performance.read-ahead  
+- performance.write-behind 
+- performance.readdir-ahead 
+- performance.io-cache 
+- performance.quick-read 
+- performance.open-behind 
+a| 
+- performance.strict-o-direct 
+|===
+
+
+.Table Title - CNS Block
+|===
+|Database | CNS Block 
+|Postgresql
+|Yes
+
+
+|MongoDB  
+|Yes
+
+|===
+
+Above mentioned Performance Translators for CNS File are already part of database profile delivered with latest CNS images. 
```

Comment:
The performance translators for the CNS file, as mentioned above, are already part of the database profile delivered with the latest CNS images.